uint32     gloveID
float32    timestemp

float32    thumb0_0
float32    thumb0_1
float32    thumb1
float32    thumb2

float32    forefinger0_0
float32    forefinger0_1
float32    forefinger1
float32    forefinger2

float32    middlefinger0_0
float32    middlefinger0_1
float32    middlefinger1
float32    middlefinger2

float32    ringfinger0_0
float32    ringfinger0_1
float32    ringfinger1
float32    ringfinger2

float32    littlefinger0_0
float32    littlefinger0_1
float32    littlefinger1
float32    littlefinger2

float32    arm0
float32    arm1_0
float32    arm1_1

geometry_msgs/Quaternion base